src/pages/Kebab/Kebab.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
 import kebabs from '../../mocks/kebab.json'
-import Product from '../Products/Product'
+import Product from '../../components/Products/Product'
+import Headline from '../../components/Headline/Headline'
 import PropTypes from 'prop-types'
-import Headline from '../Headline/Headline'
+
```

Comment:
For every product category, you use a very similar function to build its page, e.g. `Kebab.js` and `Burger.js` are nearly identical. You might want to use a utility function to streamline this and reduce the work needed to add new pages in the future.